core/app/BUILD.bazel

```diff
@@ -29,3 +31,9 @@ go_library(
         "@com_github_pkg_errors//:go_default_library",
     ],
 )
+
+gapid_version(
+    name = "version",
+    out = "default_version.go",
+    template = "default_version.go.in",
```

Comment:
It would probably be cleaner to do this with x_def stamping.

Reply:
The problem with this approach is that it modifies a source at the bottom level, which causes a cascade of rebuilds.
The stamping approach just causes a re-link at the top level.

Reply:
I'm OK with that.